pkgs/by-name/ru/rucio/package.nix

```
@@ -0,0 +1,7 @@
+{ python312Packages }:
```

Per [pkgs/by-name/README.md section "Changing implicit attribute defaults"](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults), such customisation should be specified in `pkgs/top-level/all-packages.nix` instead of hard-coding in the `<pkg>.override` interface.

```suggestion
{ python3Packages }:
```